components/core/tools/scripts/utils/build.py

```
@@ -0,0 +1,117 @@
+        cmd.append(str(num_jobs))
+    subprocess.run(cmd, check=True, cwd=str(build_dir))
+
+
+def _find_project_root() -> Path:
+    root = Path(__file__).resolve()
+    for _ in range(PROJECT_ROOT_DEPTH):
+        root = root.parent
+    return root
+
+
+def main(argv: List[str]) -> int:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)